doc/recipe/preprocessor.rst

```diff
@@ -1819,6 +1820,21 @@ cell measures or ancillary variables`.
 See also :func:`esmvalcore.preprocessor.volume_statistics`.
 
 
+``axis_statistics``
+---------------------
+
+This function operates over a given axis, and removes it of the
+output cube. 
+
+Takes arguments:
+  * axis: direction over which the statistics will be performed.
```

Comment:
Could you add a list of valid choices for axis? Or link to the relevant iris documentation?

Reply:
Sure!